File: 1758 minimum-changes-to-make-alternating-binary-string/solution.cpp
class Solution {
public:
    int minOperations(string s) {
        int ans = 0;
        int n = s.size();
        for (int i=0;i<n;i++) {
            if (((i & 1) == 0 && s[i] == '1') || ((i & 1) == 1 && s[i] == '0')) {
                ans++;
            }
        }
        return min(ans, n - ans);
    }
};

File: 1758 minimum-changes-to-make-alternating-binary-string/solution.java
class Solution {
    public int minOperations(String s) {
        int ans = 0;
        char[] arr = s.toCharArray();
        int n = arr.length;
        for (int i = 0; i < n; i++) {
            if (((i & 1) == 0 && arr[i] == '1') || ((i & 1) == 1 && arr[i] == '0')) {
                ans++;
            }
        }
        return Math.min(ans, n - ans);
    }
}
